intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -231,7 +231,11 @@ \subsection{Your data directory (\$VSC\_DATA)\label{subsec:data-directory}}
 \ifgent
 If you are running out of quota on your \$VSC\_DATA filesystem you can
 request a VO (Virtual organisation). VO members get an extra DATA filesystem: \$VSC\_DATA\_VO)
-We can give higher quota values for these filesystems. Don't hesitate to contact the \hpcTeam staff.
+We can give higher quota values for these filesystems. VO administrators can request additional quota
+at the "Edit VO" page \url{https://account.vscentrum.be/django/vo/edit}.
+Do note that additional quota is an upper bound that can be raised, but this is not
+a guarantee you will be able to use the additional quota since we do oversubscribe our storage
+to achieve optimal usage.
```

Comment:
Hmm, I find this last bit is confusing... Not sure if this is worth mentioning, since I doubt we'd let the filesystem run 100% full?

Reply:
I would just drop this last part, to be honest... In practice, we're probably going to start taking measures when available storage space becomes so limited that users are being affected, either by asking users with large volumes of data to clean up or move some of their data, or by expanding the storage.

There's a theoretical possibility that the DATA filesystem gets filled to the brim and users won't be able to use all of their quota, but I strongly doubt that will ever happen in practice, since we're keeping a close eye on it...

Reply:
Whut? Why not be 100% clear and accurate? 
let me remind you what happened last time I heard someone saying: practice learns this will never happen...

Reply:
If you want 100% accurate, you should also discuss hardware problems, possibly combined with software bugs (this may ring a bell or two), that may potentially destroy their data, or cosmic rays that may cause bit flips (it happens, see https://blogs.oracle.com/ksplice/attack-of-the-cosmic-rays).

Documentation should be clear & concise, not try to cover all potential "but-if's", especially not ones we should try hard to avoid from ever occurring (especially since operating at 90% would already be considered an important problem)...

Imho, this paragraph will only raise questions & concerns.

Reply:
@boegel That is not merely a technical possibility. It has occurred. Multiple times. Whenever we hit 85%, or higher, we've asked people to clean up their stuff. Most of those people did not have a (>85%) full VO, quota-wise. So they were not even able to use that, and since they were big consumers of storage space, we asked them first to clean up ...

Reply:
Suggestion for a better paragraph (it should be a *new* paragraph imho, not a few sentences at the end of the existing one):

```
Note that we may oversubscribe each shared filesystem, meaning that the total amount of quota across
all VOs may exceed the total size of the filesystem. Because of this, you should consider the available quota
as an upper limit of the amount of storage space that is available to you; you may not be able to use it all
if the filesystem does not provide a sufficient amount of free space.
```